test/server/cards/02_SHD/units/MigsMayfeldTriggerman.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Migs Mayfeld, Triggerman', function() {
+    integration(function(contextRef) {
+        describe('Migs Mayfeld\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['pillage', 'forced-surrender'],
+                        groundArena: [{ card: 'grogu#irresistible' }, { card: 'migs-mayfeld#triggerman' }],
+                        base: 'command-center'
+                    },
+                    player2: {
+                        hand: ['confiscate', 'daring-raid', 'krayt-dragon'],
+                        groundArena: [{ card: 'wampa', upgrades: ['shield'] }],
```

Comment:
Any reason for this to have a shield?

Reply:
I plan to test Force Throw (when implemented) with Migs and ensure that the Force Throw damage hits before the Migs trigger. Force Throw Krayt Dragon into shield should pop shield and then Migs trigger should leave Wampa with just 2 damage.